GPSTest/build.gradle

```diff
@@ -3,12 +3,12 @@ import groovy.swing.SwingBuilder
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 27
-    buildToolsVersion "27.0.3"
+    compileSdkVersion 28
+    buildToolsVersion "28.0.0"
```

Comment:
You'll need to update the build tools constant in `.travis.yml` for the build to pass on Travis CI -  it's currently failing.
https://travis-ci.org/barbeau/gpstest/builds/397615385?utm_source=github_status&utm_medium=notification